docs/get-started.md

```diff
@@ -77,6 +77,14 @@ In laravel, launch command as usual with your path:
 php artisan insights path/to/analyse
 ```
 
+## Specify composer file
```

Comment:
Should we add a badge about v2 version on this ?

Reply:
Done